src/osp/Active/SysRender.h

```diff
@@ -202,19 +237,42 @@ void SysRender::assure_draw_transforms(
     }
 }
 
+template<typename STORAGE_T, typename REFCOUNT_T>
+void remove_refcounted(
+        ActiveEnt ent, STORAGE_T &rStorage, REFCOUNT_T &rRefcount)
+{
+    if (rStorage.contains(ent))
+    {
+        auto &rOwner = rStorage.get(ent);
```

Comment:
doesn't this require two lookups?

Reply:
Assuming this looks similar to the stuff in the shaders? The shaders needed to go from ActiveEnt -> MeshGlId -> Mesh; here, it's just ActiveEnt -> Owner